content/en/docs/reference/command-line-tools-reference/kubelet.md

```diff
@@ -10,7 +10,10 @@ notitle: true
 
 
 The kubelet is the primary "node agent" that runs on each
```

Comment:
this page is generated from https://github.com/kubernetes/kubernetes/blob/master/cmd/kubelet/app/server.go#L120-L138

manual changes here will be stomped the next time it is updated. if you want to make changes, also make them in the command help

Reply:
Oops. 

I don't mind making a PR there as well, but I am concerned about putting URLs - relative or absolute - in the help in the `server.go` file. Thoughts about that?

Reply:
yeah, probably not great to deep-link help URLs from CLI output

Reply:
@kubernetes/sig-docs-pr-reviews might have suggestions

Reply:
I am going to update this to remove the elements here, and a separate PR (linked to this) for the docs in the source, as you suggested, but without actual links.

Reply:
I removed the changes to the auto-generated file. PR for CLI https://github.com/kubernetes/kubernetes/pull/73256